applications/FluidDynamicsHydraulicsApplication/python_scripts/apply_hydraulic_inlet_process.py

```diff
@@ -0,0 +1,181 @@
+from math import sqrt
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsHydraulicsApplication as KratosFluidHydraulics
+from KratosMultiphysics.read_csv_table_utility import ReadCsvTableUtility
+import KratosMultiphysics.kratos_utilities as KratosUtils
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyHydraulicInletProcess(Model, settings["Parameters"])
```

Comment:
Though this works, I'd start using the registry.

Reply:
Working on